qgroundcontrol.pro

```diff
@@ -49,6 +49,7 @@ LinuxBuild {
 WindowsBuild {
     RC_ICONS = resources/icons/qgroundcontrol.ico
     CONFIG += resources_big
+    CONFIG += console
```

Comment:
Is this just some temporary WIP thing? This shouldn't be added.

Reply:
Yes it's temporary for testing :+1:  the commit message is `wip: add console to windows build`

The appveyor artifact is failing to launch without any message, so I am trying to figure out what's up.

Reply:
If you use the `--log-output` command line option. Console output will be in a file called QGCConsole.txt in the CrashLogs directory.

Reply:
Thank you!